.env.example

```diff
@@ -73,6 +73,7 @@ GENESIS_CONTRACTS_BRANCH=master# Genesis contracts branch defining the version t
 MATIC_CLI_REPO="https://github.com/maticnetwork/matic-cli.git" # repo of matic-cli to run a specific version remotely using express-cli
 MATIC_CLI_BRANCH=master # matic-cli branch used on the remote machines to start the environment
 DEVNET_BOR_USERS=ubuntu,ubuntu,ubuntu # users' names of VMs for all the nodes (comma separated). Its length must be equal to "TF_VAR_BOR_VALIDATOR_COUNT + TF_VAR_BOR_SENTRY_COUNT + TF_VAR_BOR_ARCHIVE_COUNT"
+DEVNET_BOR_FLAGS=config,config,config # Specifies how bor should be started ('config', or 'cli'), by providing cli flags, or by using toml config file (length of DEVNET_BOR_FLAGS should match the length of DEVNET_BOR_USERS)
```

Comment:
So `config` will be the default going forward, right?
What is the alternative called?

Reply:
Yes, the alternative is `cli` (mentioned in the comment).